.jenkins/pytorch/test.sh

```diff
@@ -315,9 +316,12 @@ test_rpc() {
     echo "Testing RPC C++ tests"
     # NB: the ending test_rpc must match the current function name for the current
     # test reporting process (in print_test_stats.py) to function as expected.
+    ln -sf "$TORCH_LIB_DIR"/libtorch* "$TORCH_BIN_DIR"
+    ln -sf "$TORCH_LIB_DIR"/libc10* "$TORCH_BIN_DIR"
+    ln -sf "$TORCH_LIB_DIR"/libtbb* "$TORCH_BIN_DIR"
```

Comment:
Wondering if there's way that we can notify people if they build more binaries that need to be included for test

Reply:
I am struggling with this part too, as adding sym links doesn't seem to be an intuitive way to write test (as one may expect the executable file under installed `bin` to find the shared library in `lib`). Not sure this one https://gitlab.kitware.com/cmake/community/-/wikis/doc/cmake/RPATH-handling is a good reference but I will create an issue to follow up this part